lib/locales/es.js

```diff
@@ -60,6 +61,7 @@ module.exports = {
         'mean:': 'media:',                                            // traces/box/calc.js:133
         'median:': 'mediana:',                                        // traces/box/calc.js:128
         'min:': 'mín:',                                               // traces/box/calc.js:129
+        'new text': 'nuevo texto',
```

Comment:
Grammatically, the translation should be `texto nuevo` (as is `insertar un texto nuevo`), but in software `nuevo texto` is more common (I guess this is the result of the influence of the original English).

Reply:
Insidious, isn't it 😈 

Thanks @n-riesco , I'll merge once I get tests to pass; the new `regl` code is awesome but it's wreaking havoc on CI right now...